components/ProfilePage/StyledProfileComponents.tsx

```diff
@@ -54,13 +66,40 @@ export const ProfileDisplayName = styled(Col).attrs(props => ({
     font-size: 2.75rem;
   }
 `
+
+export const ProfileDisplayNameSmall = styled(Col).attrs(props => ({
+  className: `${props.className}`
+}))`
+  margin: 0;
+  /* max-height: 108px; */
```

Comment:
```suggestion
```

Reply:
was testing to see if some pre-existing code actually did anything and forgot to clean it up; fixed